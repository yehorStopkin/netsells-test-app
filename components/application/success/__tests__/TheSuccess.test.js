const { shallowMount } = require('@vue/test-utils');
const TheSuccess = require('@/components/application/success/TheSuccess').default;

describe('TheSuccess', () => {
    test('component default', () => {
        const wrapper = shallowMount(TheSuccess);
        expect(wrapper.exists()).toBeTruthy();
        expect(wrapper.element).toMatchSnapshot();
    });
});
