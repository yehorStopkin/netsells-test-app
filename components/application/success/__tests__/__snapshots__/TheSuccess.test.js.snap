// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheSuccess component default 1`] = `
<div
  class="the-success"
>
  <div
    class="the-success__title"
  >
    Success!
  </div>
   
  <div
    class="the-success__subtitle"
  >
    Thank You
  </div>
   
  <div
    class="the-success__message"
  >
    Our team will respond to you within the next 48h
  </div>
   
  <div
    class="the-success__actions"
  >
    <div
      text="HOME"
      width="115"
    />
  </div>
</div>
`;
